Assets/Scripts/Managers/CustomerSpawnManager.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CustomerSpawnManager : MonoBehaviour
```

Comment:
Can we put `spawnFrequencyTime` and `spawnCustomerHeight` under a scriptable object named `CustomerConfiguration`? You can refer to the SO here in this class and access these fields.